protos/order_transaction.proto

```diff
@@ -0,0 +1,52 @@
+// Copyright 2018 Bitwise IO
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// -----------------------------------------------------------------------------
+
+syntax = "proto3";
+
+import "order.proto";
+
+message CreateOrderAction {
+
+	// Whether this is a buy / sell order
+	Order.Action action = 1;
+
+	// Whether this is a market / limit order
+	Order.OrderType order_type = 2;
+
+	// Unique name of the organization the order is behalf on
+	string ordering_organization_id = 3;
```

Comment:
Again, I wonder if we need this field? As this should be the id for the organization of the participant who submitted the transaction.